CHANGELOG.md

```diff
@@ -7,6 +7,12 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 
 ## [Unpublished]
 
+### Added
+
+* Option `--passwordFile` (alias `-F`) reads the password from a file
```

Comment:
For consistency this should probably be `--password-file`. Also I think `-p` is currently unused and would be more intuitive than `-F` (which we might want to use for other purposes).